gui

```diff
@@ -131,13 +389,6 @@ runonce <<"EOF"
   fi
 EOF
 
-#for old installs prior to having categories, re-run install script
-runonce <<"EOF"
-  if [ ! -e "${DIRECTORY}/data/categories" ];then
-    "${DIRECTORY}/install"
-  fi
-EOF
-
 #install dependencies
 runonce <<"EOF"
   dependencies='yad curl wget aria2 lsb-release apt-utils imagemagick bc librsvg2-bin shellcheck'
```

Comment:
```suggestion
  dependencies='yad curl wget aria2 lsb-release apt-utils imagemagick bc librsvg2-bin shellcheck wmctrl xdotool'
```

Added `wmctrl` and `xdotool` for window focusing. They just cost about 100kb of disk space.

Reply:
> Added `wmctrl` and `xdotool` for window focusing.

Done. I've also added them to the `install` script.